crates/router/src/connector/airwallex/transformers.rs

```diff
@@ -152,10 +152,10 @@ impl TryFrom<&types::PaymentsCaptureRouterData> for AirwallexPaymentsCaptureRequ
     fn try_from(item: &types::PaymentsCaptureRouterData) -> Result<Self, Self::Error> {
         Ok(Self {
             request_id: Uuid::new_v4().to_string(),
-            amount: match item.request.amount_to_capture {
-                Some(a) => Some(utils::to_currency_base_unit(a, item.request.currency)?),
-                _ => None,
-            },
+            amount: Some(utils::to_currency_base_unit(
```

Comment:
if amount in capture request is mandatory for connector can we remove Option in amount? change it in other connectors also